preview/Dockerfile

```diff
@@ -1,28 +1,34 @@
-FROM buildpack-deps:trusty-scm
+FROM buildpack-deps:jessie-scm
```

Comment:
@naamunds, Can you open a separate issue for this?  It keeps popping up and I think it is best to capture the discussion in an issue versus this PR.  I think this PR is should be simply about switching from Ubuntu to Debian.